test/meta-colony.js

```diff
@@ -343,25 +343,6 @@ contract("Meta Colony", accounts => {
       assert.equal(rootSkillChild3.toNumber(), 6);
     });
 
-    it("should NOT be able to add a child local skill more than one level from the root local skill", async () => {
```

Comment:
Why delete these tests?

Reply:
- `should NOT be able to add a child local skill more than one level from the root local skill` is already covered by `should NOT be able to add a child domain more than one level away from the root domain`
- `should NOT be able to add a child local skill to a global skill parent` - is no longer relevant. Passing in a domain id in and creating the local skills within the same call ensures we cannot add a domain against a global skill

Reply:
I was actually tempted to merge the `when adding domains in the meta colony` and `when adding domains in a regular colony` as there is nothing special in the meta colony setup as far as domains are concerned. What do you think?